lib/mayaUsd/ufe/Utils.cpp

```diff
@@ -932,14 +932,12 @@ void handleDisplayLayer(
 
 void ReplicateExtrasFromUSD::initRecursive(Ufe::SceneItem::Ptr ufeItem) const
 {
-    auto node = Ufe::Hierarchy::hierarchy(ufeItem);
-    if (!node) {
-        return;
-    }
-
-    // Go through the entire hierarchy
-    for (auto child : node->children()) {
-        initRecursive(child);
+    auto hier = Ufe::Hierarchy::hierarchy(ufeItem);
+    if (hier) {
+        // Go through the entire hierarchy
+        for (auto child : hier->children()) {
+            initRecursive(child);
+        }
     }
```

Comment:
I think that it should process the input scene item even if there is no hierarchy (I did the same for my new method below).